### ISSUE TITLE
Trailing semicolons shouldn't be detected in strings

### ISSUE THREAD

**Issue by etiennebacher** [etiennebacher/flint#27]
``` r
flint::lint_text("'hi there;'")
#> Original code: hi there; 
#> Suggestion: Trailing semicolons are not needed.
```

**Comment by etiennebacher** [comment]
Fixed in 61ade70cfe763c09f3b25a5eb1666ae1904bb7b2
